task1.py

```diff
@@ -0,0 +1,5 @@
+n =int(input())
+a = n//60
+b = n - a*60
+if a>24: a = a%24
```

Comment:
идея должна такое подчеркивать, следи за кодстайлом
```suggestion
if a > 24: a = a % 24
```